src/main/java/thePackmaster/cards/coresetpack/DestinyDraw.java

```diff
@@ -22,7 +23,7 @@ public DestinyDraw() {
     }
 
     public void use(AbstractPlayer p, AbstractMonster m) {
-        ArrayList<AbstractCardPack> choices = SpireAnniversary5Mod.getRandomPacks(true, 3);
+        ArrayList<AbstractCardPack> choices = SpireAnniversary5Mod.getRandomPacks(p.chosenClass == ThePackmaster.Enums.THE_PACKMASTER, 3);
```

Comment:
`true` here means "pick from the pool of packs for my current run", `false` means "pick from all packs".